src/main/java/io/gingersnapproject/cdc/cache/hotrod/HotRodService.java

```diff
@@ -45,16 +47,18 @@ public void init() {
       eventing.backendStartedEvent(false);
    }
 
-   public void shutdown(@Observes ShutdownEvent e) {
+   void shutdown(@Observes ShutdownEvent ev) {
       if (rcm != null) {
-         rcm.stopAsync().whenComplete((ignore, t) -> {
+         shutdown = true;
+         RemoteCacheManager s = rcm;
+         rcm = null;
+         s.stopAsync().whenComplete((ignore, t) -> {
```

Comment:
It is possible that we shut down the manager _before_ writing the offset, which will cause an exception during termination.

Reply:
Created #58